submissions/Lu-sure/memory-pair-game/style.css

```diff
@@ -0,0 +1,143 @@
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: content-box;
+  font-family: 'Langar', cursive;
+  font-size: 18px;
+}
+
+.navig {
+  background-color:  #53709b;
+  min-height: 70px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.new-game-button {
+  font-family: 'Langar', cursive;
+  font-size: 1.5em;
+  border: solid 2px rgb(45, 19, 94);
+  border-radius: 5px;
+  color:  rgb(45, 19, 94);;
+}
+
+.board {
+  background-color: rgb(187, 196, 245);
+  max-width: 800px;
+  height: 620px;
+  margin: 0 auto;
+  box-shadow: 2px 4px 10px rgba(0, 0, 0, 0.15);
+  display: flex;
+  justify-content: space-between;
+  align-content: space-between;
+  flex-wrap: wrap;
+  padding: 20px;
+}
+
+.card {
+  height: 190px;
+  width: 175px;
+  background-color: transparent;
+  justify-content: center;
+  border: solid 2px  rgb(140, 151, 212);
+  perspective: 1000px;
+}
+
+.card:hover,
+.new-game-button:hover {
+  cursor: pointer;
+}
+
+.card-inner {
+  background-color: white;
```

Comment:
I noticed that this style (`background-color: white;`) broke the app in Safari. Remove it, please

Reply:
Removed. Check if it works properly now.